doc/tutorials/computing-product-analytics.md

```diff
@@ -0,0 +1,258 @@
+---
+jupytext:
+  notebook_metadata_filter: myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Conduct Product Analysis Using SQL
+
++++
+
+Product analytics is a common analysis. This tutorial will provide a scenario for an e-commerce website and shows you how to conduct product analysis using SQL.
+
++++
+
+## Growth and Retention
+
++++
+
+Growth rate and user retention are some common metrics used in product analytics. Suppose that we have devleoped an app for Ploomber and want to see whether the app is popular. Here is some data we collected.
+
++++
+
+### Dataset Description
+
+For simplicity, we just use some small-scale randomly generated data. The dataset **user_activity** is consists of three columns: 
```

Comment:
> For simplicity, we just use some small-scale randomly generated data.

This sentence is redundant, suggest to state something along the lines:

"For the purpose of this tutorial the data set below is used"